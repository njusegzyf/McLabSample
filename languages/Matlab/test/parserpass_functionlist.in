function y = fcn1(x)
  y = 1;
end function [y] = fcn2(), y = 2;
end
function [y, z] = fcn3(a)
  y = a + 1;
  function q = fcn4
    q = 3
    %comment1
  end
  z = y; function r = fcn5
    %comment2
    r = 7
  end y = z;
end
function fcn6
  %%helpcomment1
end
%extracomment