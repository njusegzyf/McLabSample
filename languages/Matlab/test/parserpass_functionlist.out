>>>> Destination Language -> Source Language
[1, 1] -> (1, 1)
[1, 2] -> (1, 2)
[1, 3] -> (1, 3)
[1, 4] -> (1, 4)
[1, 5] -> (1, 5)
[1, 6] -> (1, 6)
[1, 7] -> (1, 7)
[1, 8] -> (1, 8)
[1, 9] -> (1, 9)
[1, 10] -> (1, 10)
[1, 11] -> (1, 11)
[1, 12] -> (1, 12)
[1, 13] -> (1, 13)
[1, 14] -> (1, 14)
[1, 15] -> (1, 15)
[1, 16] -> (1, 16)
[1, 17] -> (1, 17)
[1, 18] -> (1, 18)
[1, 19] -> (1, 19)
[1, 20] -> (1, 20)
[1, 21] -> (1, 21)
[2, 1] -> (2, 1)
[2, 2] -> (2, 2)
[2, 3] -> (2, 3)
[2, 4] -> (2, 4)
[2, 5] -> (2, 5)
[2, 6] -> (2, 6)
[2, 7] -> (2, 7)
[2, 8] -> (2, 8)
[2, 9] -> (2, 9)
[3, 1] -> (3, 1)
[3, 2] -> (3, 2)
[3, 3] -> (3, 3)
[3, 4] -> (3, 3)
[4, 1] -> (3, 3)
[5, 1] -> (3, 5)
[5, 2] -> (3, 6)
[5, 3] -> (3, 7)
[5, 4] -> (3, 8)
[5, 5] -> (3, 9)
[5, 6] -> (3, 10)
[5, 7] -> (3, 11)
[5, 8] -> (3, 12)
[5, 9] -> (3, 13)
[5, 10] -> (3, 14)
[5, 11] -> (3, 15)
[5, 12] -> (3, 16)
[5, 13] -> (3, 17)
[5, 14] -> (3, 18)
[5, 15] -> (3, 19)
[5, 16] -> (3, 20)
[5, 17] -> (3, 21)
[5, 18] -> (3, 22)
[5, 19] -> (3, 23)
[5, 20] -> (3, 24)
[5, 21] -> (3, 25)
[5, 22] -> (3, 26)
[5, 23] -> (3, 4)
[5, 24] -> (3, 4)
[6, 1] -> (3, 27)
[6, 2] -> (3, 28)
[6, 3] -> (3, 29)
[6, 4] -> (3, 30)
[6, 5] -> (3, 31)
[6, 6] -> (3, 32)
[6, 7] -> (3, 33)
[6, 8] -> (3, 34)
[7, 1] -> (4, 1)
[7, 2] -> (4, 2)
[7, 3] -> (4, 3)
[7, 4] -> (4, 3)
[8, 1] -> (4, 3)
[9, 1] -> (5, 1)
[9, 2] -> (5, 2)
[9, 3] -> (5, 3)
[9, 4] -> (5, 4)
[9, 5] -> (5, 5)
[9, 6] -> (5, 6)
[9, 7] -> (5, 7)
[9, 8] -> (5, 8)
[9, 9] -> (5, 9)
[9, 10] -> (5, 10)
[9, 11] -> (5, 11)
[9, 12] -> (5, 12)
[9, 13] -> (5, 13)
[9, 14] -> (5, 14)
[9, 15] -> (5, 15)
[9, 16] -> (5, 16)
[9, 17] -> (5, 17)
[9, 18] -> (5, 18)
[9, 19] -> (5, 19)
[9, 20] -> (5, 20)
[9, 21] -> (5, 21)
[9, 22] -> (5, 22)
[9, 23] -> (5, 23)
[9, 24] -> (5, 24)
[9, 25] -> (5, 25)
[9, 26] -> (5, 26)
[10, 1] -> (4, 4)
[11, 1] -> (4, 1)
[12, 1] -> (6, 1)
[12, 2] -> (6, 2)
[12, 3] -> (6, 3)
[12, 4] -> (6, 4)
[12, 5] -> (6, 5)
[12, 6] -> (6, 6)
[12, 7] -> (6, 7)
[12, 8] -> (6, 8)
[12, 9] -> (6, 9)
[12, 10] -> (6, 10)
[12, 11] -> (6, 11)
[12, 12] -> (6, 12)
[12, 13] -> (6, 13)
[13, 1] -> (7, 1)
[13, 2] -> (7, 2)
[13, 3] -> (7, 3)
[13, 4] -> (7, 4)
[13, 5] -> (7, 5)
[13, 6] -> (7, 6)
[13, 7] -> (7, 7)
[13, 8] -> (7, 8)
[13, 9] -> (7, 9)
[13, 10] -> (7, 10)
[13, 11] -> (7, 11)
[13, 12] -> (7, 12)
[13, 13] -> (7, 13)
[13, 14] -> (7, 14)
[13, 15] -> (7, 15)
[13, 16] -> (7, 16)
[13, 17] -> (7, 17)
[13, 18] -> (7, 18)
[13, 19] -> (7, 19)
[13, 20] -> (7, 20)
[14, 1] -> (8, 1)
[14, 2] -> (8, 2)
[14, 3] -> (8, 3)
[14, 4] -> (8, 4)
[14, 5] -> (8, 5)
[14, 6] -> (8, 6)
[14, 7] -> (8, 7)
[14, 8] -> (8, 8)
[14, 9] -> (8, 9)
[14, 10] -> (8, 10)
[15, 1] -> (9, 1)
[15, 2] -> (9, 2)
[15, 3] -> (9, 3)
[15, 4] -> (9, 4)
[15, 5] -> (9, 5)
[15, 6] -> (9, 6)
[15, 7] -> (9, 7)
[15, 8] -> (9, 8)
[15, 9] -> (9, 9)
[15, 10] -> (9, 10)
[15, 11] -> (9, 11)
[15, 12] -> (9, 12)
[15, 13] -> (9, 13)
[15, 14] -> (9, 14)
[16, 1] -> (10, 1)
[16, 2] -> (10, 2)
[16, 3] -> (10, 3)
[16, 4] -> (10, 4)
[16, 5] -> (10, 5)
[16, 6] -> (10, 6)
[17, 1] -> (11, 1)
[17, 2] -> (11, 2)
[17, 3] -> (11, 3)
[17, 4] -> (11, 4)
[17, 5] -> (11, 5)
[17, 6] -> (11, 6)
[17, 7] -> (11, 7)
[17, 8] -> (11, 8)
[17, 9] -> (11, 9)
[17, 10] -> (11, 10)
[17, 11] -> (11, 11)
[17, 12] -> (11, 12)
[17, 13] -> (11, 13)
[17, 14] -> (11, 14)
[17, 15] -> (11, 15)
[17, 16] -> (11, 16)
[17, 17] -> (11, 17)
[17, 18] -> (11, 18)
[17, 19] -> (11, 19)
[17, 20] -> (11, 20)
[17, 21] -> (11, 21)
[17, 22] -> (11, 22)
[17, 23] -> (11, 23)
[17, 24] -> (11, 24)
[17, 25] -> (11, 25)
[17, 26] -> (11, 26)
[17, 27] -> (11, 27)
[18, 1] -> (12, 1)
[18, 2] -> (12, 2)
[18, 3] -> (12, 3)
[18, 4] -> (12, 4)
[18, 5] -> (12, 5)
[18, 6] -> (12, 6)
[18, 7] -> (12, 7)
[18, 8] -> (12, 8)
[18, 9] -> (12, 9)
[18, 10] -> (12, 10)
[18, 11] -> (12, 11)
[18, 12] -> (12, 12)
[18, 13] -> (12, 13)
[18, 14] -> (12, 14)
[19, 1] -> (13, 1)
[19, 2] -> (13, 2)
[19, 3] -> (13, 3)
[19, 4] -> (13, 4)
[19, 5] -> (13, 5)
[19, 6] -> (13, 6)
[19, 7] -> (13, 7)
[19, 8] -> (13, 8)
[19, 9] -> (13, 9)
[19, 10] -> (13, 10)
[20, 1] -> (14, 1)
[20, 2] -> (14, 2)
[20, 3] -> (14, 3)
[20, 4] -> (14, 4)
[20, 5] -> (14, 5)
[20, 6] -> (14, 6)
[20, 7] -> (14, 7)
[20, 8] -> (14, 8)
[20, 9] -> (14, 9)
[20, 10] -> (14, 10)
[20, 11] -> (14, 11)
[20, 12] -> (14, 12)
[20, 13] -> (14, 13)
[21, 1] -> (15, 1)
[21, 2] -> (15, 2)
[21, 3] -> (15, 3)
[21, 4] -> (15, 3)
[22, 1] -> (15, 3)
[23, 1] -> (16, 1)
[23, 2] -> (16, 2)
[23, 3] -> (16, 3)
[23, 4] -> (16, 4)
[23, 5] -> (16, 5)
[23, 6] -> (16, 6)
[23, 7] -> (16, 7)
[23, 8] -> (16, 8)
[23, 9] -> (16, 9)
[23, 10] -> (16, 10)
[23, 11] -> (16, 11)
[23, 12] -> (16, 12)
[23, 13] -> (16, 13)
[23, 14] -> (16, 14)
[24, 1] -> (15, 4)
[25, 1] -> (15, 1)
[26, 1] -> (17, 1)
[26, 2] -> (17, 2)
[26, 3] -> (17, 3)
[26, 4] -> (17, 4)
[26, 5] -> (17, 5)
[26, 6] -> (17, 6)
[26, 7] -> (17, 7)
[26, 8] -> (17, 8)
[26, 9] -> (17, 9)
[26, 10] -> (17, 10)
[26, 11] -> (17, 11)
[26, 12] -> (17, 12)
[26, 13] -> (17, 13)
[26, 14] -> (17, 14)
[26, 15] -> (17, 15)
[26, 16] -> (17, 16)
[26, 17] -> (17, 17)
[27, 1] -> (18, 1)
[27, 2] -> (18, 2)
[27, 3] -> (18, 3)
[27, 4] -> (18, 4)
[28, 1] -> (19, 1)
[28, 2] -> (19, 2)
[28, 3] -> (19, 3)
[28, 4] -> (19, 4)
[28, 5] -> (19, 5)
[28, 6] -> (19, 6)
[28, 7] -> (19, 7)
[28, 8] -> (19, 8)
[28, 9] -> (19, 9)
[28, 10] -> (19, 10)
[28, 11] -> (19, 11)
[28, 12] -> (19, 12)
[28, 13] -> (19, 13)
>>>> Translated Text
function y = fcn1(x)
  y = 1;
end

function [y] = fcn2(), 
 y = 2;
end

function [y, z] = fcn3(a)


  y = a + 1;
  function q = fcn4
    q = 3
    %comment1
  end
  z = y; function r = fcn5
    %comment2
    r = 7
  end y = z;
end

function fcn6


  % helpcomment1
end
%extracomment